### ISSUE TITLE
found virus in .zip thru virustotal.

### ISSUE THREAD

**Issue by TankertHogens** [OnlyAstaaa/Worm-GPT#3]

**Comment by noorali5780** [comment]
so you mean. This totally a malware itself. I guess, I will run it in sandbox to see if it works or just a scam

**Comment by McStick0** [comment]
> so you mean. This totally a malware itself. I guess, I will run it in sandbox to see if it works or just a scam

what result?

**Comment by darkmoney1971** [comment]
`No...  Is not true... It' s be like similar... But tis work different...`

**Comment by jubayersimanto20-sudo** [comment]
No...  Is not true... It' s be like similar... But tis work different...Duplicate of #
